tools/clang/lib/SPIRV/InitListHandler.h

```diff
@@ -72,8 +73,8 @@ namespace spirv {
 /// If the composite type is vector or matrix, we decompose() it into scalars as
 /// explained above. If it is a struct or array type, the element type is not
 /// guaranteed to be scalars. But still, we need to split them into their
-/// elements. For such cases, we create faux MemberExpr or ArraySubscriptExpr
-/// AST nodes for all the elements and push them into the initializers queue.
+/// elements. For such cases, we create OpCompositeExtract SPIRV instructions
```

Comment:
nit: We always use `SPIR-V` consistently everywhere. No `SPIRV`, no `spirv`, no `spir-v`, no `Spir-V`, etc.

Reply:
done